File: wellness habits.py
def freetime():
    prefences = input("To recharge what do you prefer to do; exercise, socialise or relaxing : ")
    duration = input("How much work load do you have today; heavy, meduim, light : ")
    if prefences == "exercise":
        exercise(duration)
    elif prefences == "socialise":
        socialise(duration)
    elif prefences == "relaxing":
        relaxing(duration)
    else:
        print("error")
        print("please type in one of the options given")
        freetime()
        
def exercise(duration):
    if duration == "heavy":
        print("Take 20 minutes to do yoga, a 15 minute run")
        print("Dont forget to take at least 15 minutes to do exercise you enjoy")
        exit()
    elif duration == "medium":
        print("Take 40 mins to stretch and ride a bike")
        print("Dont forget to take at least 30 minutes to do an exercise you enjoy")
        exit()
    elif duration == "light":
        print("Take an hour walk around the city, take an hour zumba class")
        print("Dont forget to take at least 45 minutes to exercise")
        exit()
    else:
        print("error")
        print("please type in one of the option given")
        freetime()
def socialise(duration):
    if duration == "heavy":
        print("Take 20 minutes to go grab a coffee ,tea etc with a friend or call a friend")
        print("Dont forget to take at least 15 minutes to take a break and refresh your mind")
        exit()
    elif duration == "medium":
        print("Take 40 mins to go to a park with a friend and talk or video call a friend")
        print("Dont forget to take at least 30 minutes to take a break and refresh your mind")
        exit()
    elif duration == "light":
        print("Take an hour walk with a friend around the city, or take an hour to eat with a friend")
        print("Dont forget to take at least 45 minutes to take a break and refresh your mind")
        exit()
    else:
        print("error")
        print("please type in one of the option given")
        freetime()
def relaxing(duration):
    if duration == "heavy":
        print("Take 20 read a book, a 15 meditation session")
        print("Dont forget to take at least 15 minutes to give your eyes a break from looking at the screen")
        exit()
    elif duration == "medium":
        print("Take 40 mins to cook some food or 35 minutes drawing")
        print("Dont forget to take at least 30 minutes to give your eyes a break from looking at the screen")
        exit()
    elif duration == "light":
        print("Take an hour to paint, take an hour to solve puzzles e.g sudoku")
        print("Dont forget to take at least 45 minutes to give your eyes a break from looking at the screen")
        exit()
    else:
        print("error")
        print("please type in one of the option given")
        freetime()
freetime()

File: wellness habits.js
function answerQuiz() {
    

    var prefList = document.getElementsByName("pref");
    var durList = document.getElementsByName("work");
    var pref = 0;
    var dur = 0;

    for (var i = 0; i < prefList.length; i++) {
        if (prefList[i].checked) {
            pref = prefList[i].value;
        }
        if (durList[i].checked) {
            dur = durList[i].value;
        }
    }
    var message;
    if (pref == "exercise" || pref == "social" || pref == "relax") {
        if (dur == "light" || dur == "medium" || dur == "heavy") {
            document.getElementById("quiz").style.display = "none";
            message = freetime(pref, dur);
            document.getElementById("reset").style.display = "block";
        } else {
            message = "You must select a duration";
        }
    } else {
        message = "You must select a preference";
    }
   document.getElementById("result").innerHTML = message;

   
}

function resetQuiz() {
    document.getElementById("quiz").style.display = "block";  
    document.getElementById("quiz").reset();
    document.getElementById("reset").style.display = "none";
    
    var message = "";
    document.getElementById("result").innerHTML = message;

}

function freetime(prefences, duration){
    if (prefences == "exercise")
    {
        return exercise(duration)
    }
    else if (prefences == "social")
    {
        return socialise(duration)
    }
    else if (prefences == "relax")
    {
        return relaxing(duration)
    }
    else
    {
        return "error"
    }
}       


function exercise(duration)
{
    if (duration == "heavy")
    {
        return "Don't forget to take at least 15 minutes to do exercise you enjoy<br> What about a quick run or doing 20 minutes of yoga?"
    }
    else if (duration == "medium")
    {
        return "Don't forget to take at least 30 minutes to do exercise you enjoy<br> How about taking 40 minutes to do some stretches and go on a bike ride?"        
    }
    else if (duration == "light")
    {
        return "Don't forget to take at least 45 minutes to do exercise you enjoy<br> Maybe take an hour long walk or do an exercise class?"        
    }
    else
    {
        return "error2"
    }
}

function socialise(duration)
{
    if (duration == "heavy")
    {
        return "Don't forget to take at least 15 minutes to take a break and refresh your mind<br> Why don't you take 20 minutes to call a friend or go grab a coffee with them?";
        
    }
    else if (duration == "medium")
    {
        return "Don't forget to take at least 30 minutes to take a break and refresh your mind<br> You could catch up with a friend - what about going for a walk around the park or doing a video call?"
        
    }
    else if (duration == "light")
    {
        return "Don't forget to take at least 45 minutes to take a break and refresh your mind <br> Maybe take a long walk or have a meal with someone?";
        
    }
    else
    {
        return "error 3"
    }
}
function relaxing(duration)
{
    if (duration == "heavy")
    {
        return "Don't forget to take at least 15 minutes to give your eyes a break from screens <br> What about reading a book for a bit or doing some meditation?";
        
    }
    else if (duration == "medium")
    {
        return "Don't forget to take at least 30 minutes to give your eyes a break from screens <br> Try cooking a nice meal or doing some art for a bit";
        
    }
    else if (duration == "light")
    {
        return "Don't forget to take at least 45 minutes to give your eyes a break from screens <br> Maybe spend a while painting or drawing, or how about solving some puzzles?";
        
    }
    else
    {
        return "error4";
    }
}

